Shaders/CrossHair.fx

```diff
@@ -0,0 +1,139 @@
+/*
+Copyright (c) 2018 Jacob Maximilian Fober
+
+This work is licensed under the Creative Commons 
+Attribution-ShareAlike 4.0 International License. 
+To view a copy of this license, visit 
+http://creativecommons.org/licenses/by-sa/4.0/.
+*/
+
+// CrossHair PS
```

Comment:
I really don't want to add a crosshair effect to the official shader repository. Please remove it from the pull request.

Reply:
No problem :)
I deleted it.